extensions/wikia/Search/classes/Test/HooksTest.php

```diff
@@ -77,6 +77,10 @@ public function testOnArticleDeleteComplete() {
 		$mockUser = $this->getMockBuilder( 'User' )
 		                 ->disableOriginalConstructor()
 		                 ->getMock();
+		$mockTitle = $this->getMockBuilder( 'Title' )
+							->disableOriginalConstructor()
+							->setMethods( array( 'getNamespace' ) )
+							->getMock();
 		$mockHooks = $this->getMock( 'Wikia\Search\Hooks', null );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Wikia\Search\Hooks" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)